cluster/addons/README.md

```diff
@@ -1,11 +1,26 @@
 # Cluster add-ons
 
 Cluster add-ons are Services and Replication Controllers (with pods) that are
-shipped with the kubernetes binaries and whose update policy is also consistent
-with the update of kubernetes cluster.
+shipped with the Kubernetes binaries and are considered an inherent part of the
+Kubernetes clusters. The add-ons are visible through the API (they can be listed
+using ```kubectl```), but manipulation of these objects is discouraged because
+the system will bring them back to the original state, in particular:
+* if an add-on is stopped, it will be restarted automatically
```

Comment:
I haven't been following the progress on upgrades closely.  Which changes added this behavior?